frontend/src/metabase/redux/uploads.ts

```diff
@@ -44,59 +44,58 @@ export const hasActiveUploads = (state: State) =>
 
 export const uploadFile = createThunkAction(
   UPLOAD_FILE_TO_COLLECTION,
-  (file: File, collectionId: CollectionId) =>
-    async (dispatch: Dispatch, getState: GetState) => {
-      const id = Date.now();
+  (file: File, collectionId: CollectionId) => async (dispatch: Dispatch) => {
```

Comment:
This is just removing `getState: GetState`

Reply:
Probably in this particular case for the sake of readability (and less meaningless changes), it's better to make it `_getState`?

Reply:
There are many places that we use either
- `(dispatch: Dispatch)` or
- `dispatch =>`

You can search by both strings to see those. I don't think leaving `getState` is a good idea if we're not going to use it otherwise, we'll have a ton of unused `getState` everywhere.